spec/index.html

```diff
@@ -4835,19 +4827,19 @@ <h4>Effective Boolean Value (EBV)</h4>
             rules reflect the rules for <code>fn:boolean</code> applied to the argument types present
             in SPARQL queries:</p>
           <ul>
-            <li>The EBV of any literal whose type is <code>xsd:boolean</code> or <span class=
-                                                                                       "type">numeric</span> is false if the lexical form is not valid for that datatype (e.g.
-              "abc"^^xsd:integer).</li>
+            <li>The EBV of any literal whose type is <code>xsd:boolean</code> or 
+              <span class="type">numeric</span> is false if the lexical form is not valid for that
+              datatype (e.g. "abc"^^xsd:integer).</li>
```

Comment:
```suggestion
              datatype (e.g., `"abc"^^xsd:integer`).</li>
```

Reply:
Extra comma?

Reply:
Not extra. Correct. `e.g.` and `i.e.` should always be followed by, and usually (not when starting a parenthetical, among other exceptions) preceded by, a comma. If you're allergic to the comma here, you could change `e.g.` to `like`, for instance.

Reply:
Disagree - the parentheses suffice and also e.g. goes in the subclause. And it's the same elsewhere.

The change here is about formatting the HTML attributes and some sorting out long lines.

This is holding up this PR so I have rhanged the text completely.